src/game-of-life/hello-world.md

```diff
@@ -86,9 +78,8 @@ extern {
 
 #[wasm_bindgen]
 pub fn greet() {
-    alert("Hello, wasm-game-of-life!");
+    alert("Hello, {{project-name}}!");
```

Comment:
The `{{project-name}}` should be replaced with `wasm-game-of-life` by `cargo generate`, so this part of the change should be reverted.

Reply:
Thank you for your comment. I've reverted this part.